gulpfile.js

```diff
@@ -398,7 +403,7 @@ gulp.task('heroku:dev', ['build'], function () {});
  * Task: `transifex-download`
  * Download translations from www.transifex.com
  */
-gulp.task('transifex-download', function () {
+gulp.task('transifex-download', function (gulp_cb) {
```

Comment:
ah.. the gulp documentation uses `cb`, but I've always found it confusing to have the two different callbacks when using `async`. Anyway, `done` sounds good.